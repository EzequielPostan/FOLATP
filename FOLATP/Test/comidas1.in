loadKB( Test/comidas.kb )
conclusion( LeGusta(John,Cacahuetes) )
runTo( Test/comidas1.out )
exit
