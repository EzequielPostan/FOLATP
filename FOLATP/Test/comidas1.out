{}
│
├─ {~Come(Bill, Cacahuetes)}
│ │
│ ├─ {~Come(x1, Cacahuetes), ~EstaVivo(x1)}
│ │ │
│ │ ├─ {Mata(Cacahuetes, x1), ~Come(x1, Cacahuetes)}
│ │ │ │
│ │ │ ├─ {~Comida(Cacahuetes)}
│ │ │ │ │
│ │ │ │ ├─ {~LeGusta(John, Cacahuetes)}
│ │ │ │ │
│ │ │ │ └─ {LeGusta(John, x1), ~Comida(x1)}
│ │ │ │
│ │ │ └─ {Comida(x1), Mata(x1, x2), ~Come(x2, x1)}
│ │ │
│ │ └─ {~Come(x1, x2), ~EstaVivo(x1), ~Mata(x2, x1)}
│ │
│ └─ {EstaVivo(Bill)}
│
└─ {Come(Bill, Cacahuetes)}
